src/pageLayout/containers/TreeNav.tsx

```diff
@@ -240,6 +241,22 @@ const TreeSidebar: FC<ReduxProps> = ({
                 />
               )}
             />
+            {CLOUD && isFlagEnabled('requestPoc') && (
+              <>
+                <TreeNav.SubHeading label="Useful Links" />
+                <TreeNav.SubItem
+                  id="requestpoc"
```

Comment:
minor nit: change this to `request-poc` to be more in line with testIDs. But this is extremely minor.

Reply:
good idea